.eslintrc.json

```diff
@@ -16,20 +22,51 @@
   "rules": {
     "consistent-return": "off",
     "func-names": "off",
-    "import/extensions": ["error", "ignorePackages"],
-    "import/no-named-as-default": "off",
+    "import/extensions": [
+      "error",
+      "ignorePackages"
+    ],
     "import/no-named-as-default-member": "off",
+    "import/no-named-as-default": "off",
+    "max-classes-per-file": "off",
     "no-async-promise-executor": "off",
     "no-await-in-loop": "off",
     "no-console": "off",
     "no-continue": "off",
     "no-labels": "off",
+    "no-loop-func": "off",
     "no-param-reassign": "off",
     "no-plusplus": "off",
     "no-restricted-syntax": "off",
     "no-shadow": "off",
     "no-underscore-dangle": "off",
-    "no-loop-func": "off",
-    "max-classes-per-file": "off"
+    "unicorn/filename-case": [
+      "error",
+      {
+        "case": "kebabCase"
+      }
+    ],
+    "unicorn/better-regex": "error",
+    "unicorn/escape-case": "error",
+    "unicorn/no-array-instanceof": "error",
+    "unicorn/no-console-spaces": "error",
+    "unicorn/no-hex-escape": "error",
+    "unicorn/no-new-buffer": "error",
+    "unicorn/no-zero-fractions": "error",
+    "unicorn/prefer-dataset": "error",
+    "unicorn/prefer-flat-map": "error",
+    "unicorn/prefer-includes": "error",
+    "unicorn/prefer-negative-index": "error",
+    "unicorn/prefer-node-append": "error",
+    "unicorn/prefer-node-remove": "error",
+    "unicorn/prefer-number-properties": "error",
+    "unicorn/prefer-reflect-apply": "error",
+    "unicorn/prefer-replace-all": "error",
+    "unicorn/prefer-set-has": "error",
+    "unicorn/prefer-spread": "error",
+    "unicorn/prefer-text-content": "error",
+    "unicorn/prefer-trim-start-end": "error",
+    "unicorn/prefer-type-error": "error",
+    "unicorn/throw-new-error": "error"
```

Comment:
These are all auto-fixed in this PR, and seem generally good. Happy to remove any we think inappropriate.

Reply:
I especially like the one that turns all the `-1` checks into `includes`